homing_missiles.py

```diff
@@ -109,12 +109,22 @@ def draw_entities(self, screen):
         pygame.draw.rect(screen, self.main_character_color_, main_character_rect)
         self.draw_friendly_projectiles(screen)
 
+    def determine_enemy_color(self, enemy):
+        enemy_type = enemy.get_type()
+        if enemy_type == "BasicGrunt":
+            return YELLOW
+        elif enemy_type == "NotSoBasicGrunt":
+            return WHITE
+
```

Comment:
I still think the `else` is better but that's a fair argument, I can accept that..